front/app/modules/commercial/native_surveys/admin/components/SurveyBuilderSettings/index.tsx

```diff
@@ -0,0 +1,62 @@
+import React from 'react';
+
+// styles
+import styled from 'styled-components';
+import { colors } from 'utils/styleUtils';
+
+// components
+import { Title, Box, stylingConsts } from '@citizenlab/cl2-component-library';
+import Button from 'components/UI/Button';
+import CloseIconButton from 'components/UI/CloseIconButton';
+
+// intl
+import messages from './messages';
+import { FormattedMessage } from 'utils/cl-intl';
+
+const StyledBox = styled(Box)`
+  box-shadow: -2px 0px 1px 0px rgba(0, 0, 0, 0.06);
+`;
```

Comment:
It would probably be a good idea to add this box shadow property to the component library, so we won't have to use a styled one. Maybe something we can make a note of and add in the future when we have time?

Reply:
Yeah, there are a couple of other such things that could be improved from the component library side. We could probably create task tickets for them as improvements.